src/uwtools/tests/config/test_validator.py

```diff
@@ -202,6 +213,29 @@ def test_validate_yaml_rocoto_invalid_walltime_pattern(rocoto_assets, caplog):
         assert logged(caplog, "'0:01:00' does not match '^[0-9]{2}:[0-9]{2}:[0-9]{2}$'")
 
 
+def test_validate_yaml_rocoto_invalid_dependency_bool(rocoto_assets, caplog):
```

Comment:
Last idea, do we want to place your two new test cases for `dependencies` in alphabetical order with the others I previously added? More of a question on preference. Feel free to do whatever you think looks best.

Reply:
Good point, will do!